### DISCUSSION TITLE
Is it possible to reuse the bundled python env?

### DISCUSSION THREAD

**Discussion by zerocewl** [pyinstaller/pyinstaller#6863] [Help]
Is it possible to use the python env used by the bundled app from the command line?

I'm searching for something like env `activate` to reuse some of the installed packages from outside the app.

Or is there any call to access the bundled python directly without the app e.g. without using the `app.exe`?

(Sry if this is a duplicate question, but i haven't found anything in the docs / issues.)

**Comment by rokm** [comment]
The only way to "reuse" the embedded python interpreter is via the application's executable, for example adding an `--interactive` switch that drops in you into interactive console, or `--run` switch that lets you run a custom script (provided all requirements are available in the frozen environment).
